torchvision/datasets/mnist.py

```diff
@@ -95,21 +103,14 @@ def download(self):
         if self._check_exists():
             return
 
-        # download files
-        try:
-            os.makedirs(os.path.join(self.root, self.raw_folder))
-            os.makedirs(os.path.join(self.root, self.processed_folder))
-        except OSError as e:
-            if e.errno == errno.EEXIST:
-                pass
-            else:
-                raise
+        os.makedirs(self.raw_folder, exist_ok=True)
```

Comment:
`exist_ok` is Python 3 only. Can you make this Python 2 compatible?

Reply:
How about `if not os.path.exist()`?

Reply:
Ok, will do